src/modules/ROOT/pages/changelog/202309/00.json

```diff
@@ -0,0 +1,261 @@
+      ]
+    },
+    {
+      "what": [
+        "an option to create a custom button for a record page in its page layout configurator that will be present when viewing the page layout."
+      ],
+      "why": [
+        "to provide the user with the option to create a custom button."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1484"
+      ]
+    },
+    {
+      "what": [
+        "an icon containing an exclamation mark to a page block toolbar indicating that the concrete page block is used in another page layout."
+      ],
+      "why": [
+        "to mark a page block that is already used in another page layout."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1495"
+      ]
+    },
+    {
+      "what": [
+        "the option to change the default colors of {PRODUCT_NAME} applications in {APP_NAME_ADMIN}."
+      ],
+      "why": [
+        "to provide the user with more freedom to style {PRODUCT_NAME}."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1469"
+      ]
+    },
+    {
+      "what": [
+        "a rounded circle icon that marks a configuration option as performance impacting."
+      ],
+      "why": [
+        "to mark which features would impact the performance of an application."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1490"
+      ]
+    }
+  ],
+
+  "changed": [
+    {
+      "what": [
+        "triggering of unsaved changes message after user wants to delete a modified resource like a page, namespace, module or chart."
+      ],
+      "why": [
+        "to provide a more intuitive way to navigate in {APP_NAME_COMPOSE} after trying to delete a resource."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1466"
+      ]
+    },
+    {
+      "what": [
+        "the navigation flow when switching between different {APP_NAME_COMPOSE} resources using the sidebar navigation menu.",
+        "For example, when trying to switch from one page's builder mode to another, you were redirected to the edit mode of the page."
+      ],
+      "why": [
+        "to provide a more intuitive way to navigate between {APP_NAME_COMPOSE} resources."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1457"
+      ]
+    },
+    {
+      "what": [
+        "the functionality of the delete and un-delete button by extending it to show a spinner while a resource is being deleted or un-deleted."
+      ],
+      "why": [
+        "to show the user the current state of the delete or un-delete process."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1464"
+      ]
+    },
+    {
+      "what": [
+        "the syntax of forms for creating and modifying a resource to better support mobile users in {APP_NAME_ADMIN}."
```

Comment:
I'm not sure I get what we changed; wdym by the syntax? The code? Perhaps "the form structure to better ..." would sound better

Reply:
Not sure what I wanted to say here tbh